packages/core/src/config/tax/address-based-tax-zone-strategy.ts

```diff
@@ -0,0 +1,37 @@
+import { Channel, Logger, Order, RequestContext, Zone } from '@vendure/core';
+import { TaxZoneStrategy } from '@vendure/core/dist/config/tax/tax-zone-strategy';
+
+const loggerCtx = 'AddressBasedTaxZoneStrategy';
+
+/**
+ * @description
+ * Address based {@link TaxZoneStrategy} which tries to find the applicable {@link Zone} based on the
+ * country of the billing address, or else the country of the shipping address of the Order.
+ *
+ * Returns the default {@link Channel}'s default tax zone if no applicable zone is found.
+ *
+ * :::info
+ *
+ * This is configured via `taxOptions.taxZoneStrategy = new AddressBasedTaxZoneStrategy()` in
+ * your VendureConfig.
+ *
+ * :::
+ *
+ * @docsCategory tax
+ */
```

Comment:
Is this enough to include it in the docs?

Reply:
Yes that's fine